### PR TITLE
:sparkles: IMPROVE: 1-improve-asset-management

### PR DESCRIPTION
**Pull request by Abdullah-dev1**

### CODE REVIEW

File: src/components/header/CTA.jsx [hunk 1]
```diff
@@ -1,10 +1,13 @@
-import CV from "../../assets/cv.pdf";
-
 const CTA = () => {
   return (
     <div className="cta">
-      <a href={CV} download className="btn">
-        Download CV
+      <a
+        href="https://docs.google.com/document/d/1wctCQrFqjGCS9nr7dytPnAluL8aXJZ2c/edit?usp=sharing&ouid=110828699296883501870&rtpof=true&sd=true"
```
Comment (connect-bilal) [inline src/components/header/CTA.jsx:5]: Google doc link is not public.

File: src/assets/image.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export { default as avatar1 } from './avatar1.jpeg';
+export { default as avatar2 } from './avatar2.jpeg';
+export { default as avatar3 } from './avatar3.jpeg';
+export { default as avatar4 } from './avatar4.jpeg';
+export { default as bgTexture } from './bg-texture.png';
+export { default as meAbout } from './me-about.jpeg';
+export { default as me } from './me.png';
+export { default as portfolio1 } from './portfolio1.png';
```
Comment (connect-bilal) [inline src/assets/image.js:8]: Image could be descriptive. Avatar 1, 2, 3... is not looks good. Same for other images.

**Review by connect-bilal — CHANGES_REQUESTED** [review]
Overall changes looks good. Just minor adjustments required. Nice Work!

File: src/components/header/CTA.jsx (hunk 1 above)
Comment (Abdullah-dev1) [reply src/components/header/CTA.jsx:5]: i have resolved this issue please check

**Pull request merged**